locales/cy/translation.json

```diff
@@ -371,6 +371,26 @@
         }
       }
     },
+    "drivingLicenceDetailsNotCorrect": {
+      "title": "If your driving licence details are not correct",
```

Comment:
It might be helpful to have a `TODO` next to these for when we come back and do the welsh translations to help us find it easier